packages/components/Organisms/Publication/PublicationPagination.vue

```diff
@@ -31,9 +31,11 @@
 <script>
 import RplIcon from '@dpc-sdp/ripple-icon'
 import { RplLink, RplTextLabel } from '@dpc-sdp/ripple-link'
+import deprecate from '@dpc-sdp/ripple-global/mixins/deprecate'
```

Comment:
Do we need to replace publication page template to use the new one? So we can remove the deprecated one in the future?